entries/j/justgiving.com.json

```
@@ -1,11 +1,10 @@
 {
   "JustGiving": {
     "domain": "justgiving.com",
-    "url": "https://www.justgiving.com/",
-    "contact": {
-      "facebook": "justgiving",
-      "twitter": "JGhelp"
-    },
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://community.blackbaud.com/blogs/65/7747",
```

This seems to be a more "official" documentation:
```suggestion
    "documentation": "https://justgiving-charity-support.zendesk.com/hc/en-us/articles/360019687158",
```

---

I did see that one too, but it was a bit of a weird one since when you visit support on their website, it doesn't take you to a Zendesk subdomain. Regardless, I will update the URL and submit a new pull

---

`zendesk.com` is a popular provider for customer support services. There are already 20 entries in our list that use it.

---

Ye, I use Zendesk in my work ;) Just didn't see it linked anywhere on the JustGiving site. Anyways, it's fixed and merged now 👍